l10n_ch_payment_slip_voucher/wizard/bvr_import.py

```diff
@@ -0,0 +1,165 @@
+            [('transaction_ref', '=', reference),
+             ('reconcile_id', '=', False),
+             ('account_id.type', 'in', ['receivable', 'payable']),
+             ('journal_id.type', '=', 'sale')],
+            order='date desc',
+        )
+        if not line:
+            raise exceptions.Warning(
+                "Can't find credit line for reference %s" % reference)
+        if len(line) > 1:
+            raise exceptions.Warning(
+                "Too many credit lines for reference %s" % reference)
+        if not line.partner_id:
+            raise exceptions.Warning(
+                "Can't find a partner for reference %s" % reference)
+        return line.partner_id
+
+    def _import_v11(self):
+        """Import v11 file and transfor it into vouchers
+
+        :returns: action dict
+        :rtype: dict
+        """
+        attachment_obj = self.env['ir.attachment']
+        v11_wizard = self.env['v11.import.wizard']
+        voucher_obj = self.env['account.voucher']
+        voucher_line_obj = self.env['account.voucher.line']
+        v11file = self.v11file
+        if not v11file:
+            raise exceptions.Warning(
+                _('Please select a file first!')
+            )
+        try:
+            lines = base64.decodestring(v11file).split("\r\n")
+        except ValueError as decode_err:
+            raise exceptions.Warning(
+                _('V11 file can not be decoded, '
+                  'it contains invalid caracter %s'),
+                repr(decode_err)
+            )
+        std_importer = v11_wizard.create({
+            'v11file': self.v11file,
+            'total_cost': self.total_cost,
+            'total_amount': self.total_amount,
+            })
+        records = std_importer._parse_lines(lines)
+        voucher_ids = []
+        for record in records:
+            partner = self.get_partner_from_ref(record['reference'])
+            voucher = voucher_obj.create(
+                self._build_voucher_header(partner, record))
+            voucher_line_obj.create(
+                self._build_voucher_line(partner, record, voucher.id))
+            voucher_ids.append(voucher.id)
+        for voucher_id in voucher_ids:
+            attachment_obj.create(
+                {
+                    'name': 'V11 %s' % time.strftime(
+                        "%Y-%m-%d_%H:%M:%S", time.gmtime()
+                    ),
+                    'datas': self.v11file,
+                    'datas_fname': 'BVR %s.txt' % time.strftime(
+                        "%Y-%m-%d_%H:%M:%S", time.gmtime()
+                    ),
+                    'res_model': 'account.voucher',
+                    'res_id': voucher_id,
+                },
+            )
+
+        action_res = self.env['ir.actions.act_window'].for_xml_id(
+            'account_voucher', 'action_vendor_receipt')
+        action_res[
+            'domain'
+        ] = "[('id','in', ["+','.join(map(str, voucher_ids))+"])]"
```

Comment:
you could simply provide it in python, it works too:
```
action_res['domain'] = [('id' ,'in', voucher_ids)]
```
Or with a string but simpler (`%s` will repr the list the same way):
```
action_res['domain'] = "[('id' ,'in', %s)]" % (voucher_ids,)
```